.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,78 @@
+name: Bug report
+description: Submit a bug report to help us improve
+title: "[BUG] "
+labels: ["C-bug", "S-awaiting-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We value your time and effort to submit this bug report. Here are a few things to check before clicking the submit button :)
+        1. Make sure you're on the latest version of the app.
+        2. Please try to fill this template completely to the best of your abilities. A bug report that is clear and has a reproducible example lets us get to work faster.
```

Comment:
Can we use CHat gpt to reframe the sentences bro becauses if anyone from hyps sees this it will be easily recognizable

Reply:
cool